src/run/run.c

```diff
@@ -613,6 +618,10 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_shell = true;
                         break;
 
+                case ARG_JOB_MODE:
+                        arg_job_mode = optarg;
```

Comment:
Please check the value.

Also, support "--job-mode=help", and list the possible job modes.
See e.g. https://github.com/systemd/systemd/blob/7d3ae6b21308ca8959d7c70b1f9fa9012db6ec15/src/journal/journalctl.c#L459-L463